docs/frontend.md

```diff
@@ -175,6 +175,14 @@ and independent view components, rendered server-side.
 
 For more information, refer to the [components `README.md`](../app/components/README.md).
 
+To preview components and their available options, we use [Lookbook](https://lookbook.build/) to
+generate a navigable index of our available components. These previews are available at the [`/components/` route](http://localhost:3000/components/)
+in local development, review applications, and in the `dev` environment. When adding a new component
```

Comment:
should we just link the dev environment? or are trying to minimize crawling of it?

Reply:
It was intentional to add some light friction here with the assumption that there's enough detail to help the target audience find their way to the page, without directly linking to the deployed environment. (I think we don't usually link there?)

Reply:
prob fine to leave as-is